src/include/migraphx/matcher.hpp

```diff
@@ -372,9 +372,9 @@ match::matcher_result find_match(module& modl, M&& m)
 MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_TRACE_MATCHES)
 MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_VALIDATE_MATCHES)
 
-/// Find matches for an instruction in the module
+/// Find matches for an instruction in the module for per section of matchers
 template <class Mod, class... Ms>
-void find_matches(Mod& mod, instruction_ref ins, Ms&&... ms)
+void find_matches(size_t trace_mod, Mod& mod, instruction_ref ins, Ms&&... ms)
```

Comment:
```suggestion
void find_matches(size_t trace_pass, Mod& mod, instruction_ref ins, Ms&&... ms)
```
IMO, trace_pass would be better name. `MIGRAPHX_TRACE_MATCHES` is also already matching inside the module.

Reply:
Done